distribution/docs/src/main/resources/_contents/_broker/message-ui-contents.adoc

```diff
@@ -0,0 +1,33 @@
+==== Undelivered Messages UI
+
+===== Prerequisites for Undelivered Messages UI
+
+None
+
+===== Installing and Uninstalling
+
+The Undelivered Messages UI is installed as a part of the <<_installing_and_uninstalling_message_broker_app, Message Broker App installation>>.
+
+===== Configuring
+
+Currently there are no configurations for the Undelivered Messages UI
+
+===== Using the Undeliverable Messages UI
+The Undeliverable Messages tab gives an administrator the ability to view undeliverable messages and then decide whether to resend or delete those messages.
```

Comment:
This should probably move up to line two to be the introduction to this section.